### PR TITLE
Netgear M5300 series

### PR DESCRIPTION
**Pull request by BenBE**
This includes the Netgear M5300 series of managed switches (24/48 port managed). If there's anything to change/update, drop me a note.

Unfortunately I have no images available, and while the manuals include proper front/back views of the devices, those likely conflict with this project's license, thus were not used.

### CODE REVIEW

**Comment by BenBE** [comment]
?

**Comment by Wlanfr3ak** [comment]
Checks are valid, seem like no reaction from the Team :-/

**Comment by danner26** [comment]
Sorry all, this one slipped through. Not sure what happened. I have reopened it and will review now

File: device-types/Netgear/GSM7228S.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
+---
+manufacturer: Netgear
+model: ProSafe M5300-28G (GSM7228S)
+slug: netgear-prosafe-gsm7228s
+part_number: GSM7228S
+comments: '[M5300-28G (GSM7228S) – ProSAFE 24-Port-L2+-Managed-Stackable-Gigabit-Switch](https://netgear.de/support/product/m5300-28g%20(gsm7228s)#docs)'
+u_height: 1
```
Comment (danner26) [inline device-types/Netgear/GSM7228S.yaml:7]: Hello, thank you for your contribution! In an attempt to have the most accurate and complete data we can, I have a few additional data points we would like you to add if possible.

1. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

If you have any questions, please let us know :)

File: device-types/Netgear/GSM7252PS.yaml [hunk 2]
```diff
@@ -0,0 +1,117 @@
+---
+manufacturer: Netgear
+model: ProSafe M5300-52G-POE+ (GSM7252PS)
+slug: netgear-prosafe-gsm7252ps
+part_number: GSM7252PS
+comments: '[GSM7252PS – ProSAFE 52-Port-L2-Managed-Stackable-Gigabit-Ethernet-Switch with PoE](https://netgear.de/support/product/gsm7252ps#docs)'
+u_height: 1
```
Comment (danner26) [inline device-types/Netgear/GSM7252PS.yaml:7]: Hello, thank you for your contribution! In an attempt to have the most accurate and complete data we can, I have a few additional data points we would like you to add if possible.

1. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

If you have any questions, please let us know :)

File: device-types/Netgear/GSM7252S.yaml [hunk 3]
```diff
@@ -0,0 +1,117 @@
+---
+manufacturer: Netgear
+model: ProSafe M5300-52G (GSM7252S)
+slug: netgear-prosafe-gsm7252s
+part_number: GSM7252S
+comments: '[M5300-52G (GSM7252S) – ProSAFE 48-Port-L2+-Managed-Stackable-Gigabit-Switch](https://netgear.de/support/product/m5300-52g%20(gsm7252s)#docs)'
+u_height: 1
```
Comment (danner26) [inline device-types/Netgear/GSM7252S.yaml:7]: Hello, thank you for your contribution! In an attempt to have the most accurate and complete data we can, I have a few additional data points we would like you to add if possible.

1. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

If you have any questions, please let us know :)

File: device-types/Netgear/GSM7328FS.yaml [hunk 4]
```diff
@@ -0,0 +1,69 @@
+---
+manufacturer: Netgear
+model: ProSafe GSM7328FS
+slug: netgear-prosafe-gsm7328fs
+part_number: GSM7328FS
+comments: '[GSM7328FS – ProSAFE L3-Managed-Stackable-Switch (24 SFP, 4 GbE)](https://netgear.de/support/product/gsm7328fs#docs)'
+u_height: 1
```
Comment (danner26) [inline device-types/Netgear/GSM7328FS.yaml:7]: Hello, thank you for your contribution! In an attempt to have the most accurate and complete data we can, I have a few additional data points we would like you to add if possible.

1. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

If you have any questions, please let us know :)

File: device-types/Netgear/GSM7328Sv1.yaml [hunk 5]
```diff
@@ -0,0 +1,69 @@
+---
+manufacturer: Netgear
+model: ProSafe GSM7328Sv1
+slug: netgear-prosafe-gsm7328sv1
+part_number: GSM7328Sv1
+comments: '[GSM7328Sv1 – ProSAFE L3-Managed-Stackable-Gigabit-Ethernet-Switch (24+4)](https://netgear.de/support/product/gsm7328sv1#docs)'
+u_height: 1
```
Comment (danner26) [inline device-types/Netgear/GSM7328Sv1.yaml:7]: Hello, thank you for your contribution! In an attempt to have the most accurate and complete data we can, I have a few additional data points we would like you to add if possible.

1. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

If you have any questions, please let us know :)

File: device-types/Netgear/GSM7328Sv2.yaml [hunk 6]
```diff
@@ -0,0 +1,69 @@
+---
+manufacturer: Netgear
+model: ProSafe GSM7328Sv2
+slug: netgear-prosafe-gsm7328sv2
+part_number: GSM7328Sv2
+comments: '[GSM7328Sv2 – ProSAFE L3-Managed-Stackable-Gigabit-Ethernet-Switch (24+4)](https://netgear.de/support/product/gsm7328sv2#docs)'
+u_height: 1
```
Comment (danner26) [inline device-types/Netgear/GSM7328Sv2.yaml:7]: Hello, thank you for your contribution! In an attempt to have the most accurate and complete data we can, I have a few additional data points we would like you to add if possible.

1. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

If you have any questions, please let us know :)

File: device-types/Netgear/GSM7352Sv1.yaml [hunk 7]
```diff
@@ -0,0 +1,117 @@
+---
+manufacturer: Netgear
+model: ProSafe GSM7352Sv1
+slug: netgear-prosafe-gsm7352sv1
+part_number: GSM7352Sv1
+comments: '[GSM7352Sv1 – ProSAFE L3-Managed-Stackable-Gigabit-Ethernet-Switch (48+4)](https://netgear.de/support/product/gsm7352sv1)'
+u_height: 1
```
Comment (danner26) [inline device-types/Netgear/GSM7352Sv1.yaml:7]: Hello, thank you for your contribution! In an attempt to have the most accurate and complete data we can, I have a few additional data points we would like you to add if possible.

1. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

If you have any questions, please let us know :)

File: device-types/Netgear/GSM7352Sv2.yaml [hunk 8]
```diff
@@ -0,0 +1,117 @@
+---
+manufacturer: Netgear
+model: ProSafe GSM7352Sv2
+slug: netgear-prosafe-gsm7352sv2
+part_number: GSM7352Sv2
+comments: '[GSM7352Sv2 – ProSAFE L3-Managed-Stackable-Gigabit-Ethernet-Switch (48+4)](https://netgear.de/support/product/gsm7352sv2)'
+u_height: 1
```
Comment (danner26) [inline device-types/Netgear/GSM7352Sv2.yaml:7]: Hello, thank you for your contribution! In an attempt to have the most accurate and complete data we can, I have a few additional data points we would like you to add if possible.

1. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

If you have any questions, please let us know :)

**Comment by BenBE** [comment]
Updated the weights. Also added the console port I missed to include in the first revision.

**Review by danner26 — APPROVED** [review]

**Pull request merged**